src/helm/benchmark/presentation/summarize.py

```diff
@@ -1321,6 +1369,7 @@ def run_pipeline(self, skip_completed: bool, num_instances: int) -> None:
         self.read_overlap_stats()
 
         self.write_executive_summary()
+        self.write_schema()
```

Comment:
Is there any reason this is moved down?

Reply:
Yes, `write_schema()` now needs to go after `read_runs()`, because the runs are now needed to filter out the models that get written to the schema.

Eventually, it would be good to do a refactor to change all the instance variables to local variables, which will make the inputs and outputs of each step more clear.

Reply:
Got it.  Maybe leave a comment?

Reply:
Sure, added a comment.